src/utils/api/getBlockchainTransactions.ts

```diff
@@ -30,7 +30,7 @@ async function getBlockchainTransactions(account: string): Promise<false | IGetB
     console.error("error getBlockchainTransactions():", e);
     return false;
   }
-  return result.transactions;
+  return result;
```

Comment:
would like to know why this change exists. seems sus to me.

Reply:
was part of eslint prod cleanup, when I fixed an `any` i had to make some minor adjustments to types which meant changing what I passed down